applications/app/controllers/CrosswordsController.scala

```diff
@@ -107,19 +119,7 @@ object CrosswordSearchController extends Controller with ExecutionContexts {
 
   def lookup() = Action.async { implicit request =>
     lookupForm.bindFromRequest.get match {
```

Comment:
[It is now here](https://github.com/guardian/frontend/blob/master/common/app/controllers/EmailSignupController.scala#L124)